modules/vpc/locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
+  vpc_name = "devops-vpc"
+}
```

Comment:
I don't think we need this.

Can we add `namespace` and/or `app_name` to environment variables and use from `core`? Or follow easyhotel approach where they defined them in `locals` inside of `main`:

(have a look at how they are passed to VPC resource for the name)

https://github.com/nimblehq/easyhotel-infra-aws/blob/7187b356315a2bd845e874d0c4424228981abef7/core/locals.tf#L5

Reply:
I just want to follow Nimble's convention of module that also contains the `locals.tf`, we don't have any specific of how to call the locals, so I think it's fine then 🤔
![image](https://github.com/nvminhtue/tyson_devops_ic/assets/42540264/a9350611-6885-4f12-a45f-521e5ba47957)